<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>EXL</author>
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>zUnlock-ZN5</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>556</width>
       <height>103</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>556</width>
       <height>103</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:6pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;ZN5 (T-Mobile) CG45 Patcher&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="openExternalLinks">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>File Path:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditFilePath">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Please open the *.smg file</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QFrame" name="frame_3">
          <property name="minimumSize">
           <size>
            <width>320</width>
            <height>0</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Instruction&lt;/span&gt;&lt;/p&gt;&lt;p&gt;1. &lt;a name=&quot;result_box&quot;/&gt;Install the drivers (&lt;a href=&quot;http://www.motofan.ru/files/soft/Handset_USB_Driver_32_v3.7.0.msi-266.zip&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;x32&lt;/span&gt;&lt;/a&gt;, &lt;a href=&quot;http://www.motofan.ru/files/soft/Handset_USB_Driver_x64_v3.7.0.zip&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;x64&lt;/span&gt;&lt;/a&gt;) for your phone;&lt;br/&gt;2. Install the &lt;a href=&quot;https://www.google.com/search?q=download+RSD+Lite&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;RSDLite 3.x.x&lt;/span&gt;&lt;/a&gt; or higher for flashing;&lt;br/&gt;3. Install &lt;a href=&quot;http://forum.motofan.ru/index.php?showtopic=155910&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;MMBackup&lt;/span&gt;&lt;/a&gt; and dump your SW; &lt;br/&gt;4. Get the NoRSA bootloader: &lt;a href=&quot;http://forum.motofan.ru/index.php?act=Attach&amp;amp;type=post&amp;amp;id=204833&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;ZN5&lt;/span&gt;&lt;/a&gt; &amp;amp; &lt;a href=&quot;http://forum.motofan.ru/index.php?act=Attach&amp;amp;type=post&amp;amp;id=223937&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;ZN5 T-Mobile&lt;/span&gt;&lt;/a&gt;;&lt;br/&gt;5. Flash NoRSA bootloader;&lt;br/&gt;6. Use &lt;a href=&quot;http://forum.motofan.ru/index.php?act=Attach&amp;amp;type=post&amp;amp;id=155007&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;SBF-Recalc&lt;/span&gt;&lt;/a&gt; for split SBF to SMG files;&lt;br/&gt;7. Patch your CG45 with this program;&lt;br/&gt;8. Build SBF with patched CG45 by SBF-Recalc;&lt;br/&gt;9. Flash &lt;a name=&quot;result_box&quot;/&gt;the resulting SBF-file;&lt;br/&gt;10. Enjoy!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
             <property name="openExternalLinks">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="DropArea" name="labelDropFile">
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>175</height>
           </size>
          </property>
          <property name="acceptDrops">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Please &lt;br/&gt;Drag'n'Drop &lt;br/&gt;CG45 SMG file &lt;br/&gt;in this area&lt;br/&gt;for patch&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QFrame" name="frame_4">
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;About&lt;br/&gt;&lt;br/&gt;&lt;/span&gt;&lt;a href=&quot;https://github.com/EXL/zUnlock-ZN5&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;Source code&lt;/span&gt;&lt;/a&gt;&lt;br/&gt;&lt;a href=&quot;mailto:exl@bk.ru&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;exl@bk.ru&lt;/span&gt;&lt;/a&gt;&lt;br/&gt;EXL, 2014&lt;br/&gt;MotoFan.Ru&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
               <property name="openExternalLinks">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>18</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonOpenCG45">
            <property name="text">
             <string>&amp;Open File...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonAbout">
            <property name="text">
             <string>&amp;Help...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Log Output</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>128</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Monospace</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="lineWrapMode">
         <enum>QTextEdit::NoWrap</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>textBrowser</zorder>
     <zorder>progressBar</zorder>
     <zorder>label_4</zorder>
     <zorder>frame_3</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DropArea</class>
   <extends>QLabel</extends>
   <header>DropArea.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
